Pod/Classes/WPMediaCollectionViewCell.m

```diff
@@ -193,8 +193,17 @@ - (void)fetchAssetImage
     requestKey = [_asset imageWithSize:requestSize completionHandler:^(UIImage *result, NSError *error) {
         if (error) {
             self.image = nil;
+            self.imageView.contentMode = UIViewContentModeCenter;
+            self.imageView.backgroundColor = [UIColor blackColor];
+            if (_asset.assetType == WPMediaTypeImage) {
+                self.image = [WPMediaPickerResources imageNamed:@"camera" withExtension:@"png"];
+            } else if (_asset.assetType == WPMediaTypeVideo) {
+                self.image = [WPMediaPickerResources imageNamed:@"video" withExtension:@"png"];
+            }
             return;
         }
+        self.imageView.contentMode = UIViewContentModeScaleAspectFill;
```

Comment:
I wonder if we should do this either in `prepareForReuse` or `setImage:animated:` instead?